api/pom.xml

```diff
@@ -94,6 +94,12 @@
             <version>2.1.0</version>
         </dependency>
 
+        <dependency>
+            <groupId>com.azure</groupId>
```

Comment:
My biggest concern here is creating a precedent of including third-party arbitrary dependencies for vendor-specific things. Once we include whole google cloud SDK and bunch of other things next thing we know our image size is quadrupled.  @kafbat/backend @germanosin  thoughts?

Reply:
This precedent was already established with AWS. I can hand code a client if needed, though.

![image](https://github.com/user-attachments/assets/cb1f01b6-2762-427b-a989-f3065fe8e6cd)

Reply:
It doesn't pull in a whole lot of transitive dependencies either https://mvnrepository.com/artifact/com.azure/azure-identity/1.13.2